### ISSUE TITLE
Where are we supposed to be getting this homework link?

### ISSUE THREAD

**Issue by jdstead** [uwhpsc-2016/syllabus#18]
The homework PDF states that we will receive an email with a link to create the repository. I have not received any email with a link except for the one explaining the issues. That link doesn't take me anywhere except for an error page.

**Comment by ckchmod** [comment]
Chris sent out an e-mail saying the repo is broken on Github's end. He suggested contacting and e-mailing them. I've filed an issue myself, and just patiently waiting for them to respond. I assume most everyone else doesn't have the repo yet.

Update: It works for me now.

**Comment by Marisa67** [comment]
I got this from GitHub, I still cannot get the repo.

If you are still experiencing this issue please have your instructor log out of Classroom for GitHub, then log back in. If the issue persists then your instructor should open an issue here:

https://github.com/education/classroom/issues

Cheers,
Scott

**Comment by LaTournesol** [comment]
I got the same response as Marisa67, and I still cannot get the repo either.

**Comment by ckchmod** [comment]
The link from the very first e-mail (classroom invitation) about Homework 1 doesn't work? The link at the bottom of the e-mail...

**Comment by krvikassingh** [comment]
I too got same response as Marisa67 and LaTournesol and can't get the repo.

**Comment by yusufmansour** [comment]
I got the same response as well from GitHub Education Staff.. can't get to the repo

**Comment by jdstead** [comment]
Right, the very first email, at least for me was just the explanation that there was a glitch. It has an example of the link I would have gotten, a generic if you will. Then, in the "How to fix it email" I just cut/pasted that link which I assume is what I was supposed to get initially. And then sent in that help request verbatim. But I am not sure that this is the right link to be using. I was just curious if the emailed link that the assignment says we all got somehow didn't make it to me.

**Comment by ckchmod** [comment]
Not the one that explains the glitch. This one: https://classroom.github.com/assignment-invitations/d3d9349953aac7b4b768350750cc250c

**Comment by jdstead** [comment]
Ok good, that is the one that I ended up copying and pasting from the 'how to fix it' email. That is what I assumed I was supposed to have gotten initially. I didn't get it which is the source of my confusion. 
Thanks

**Comment by cswiercz** [comment]
@jdstead The announcement with the link to obtain the repo is here: https://canvas.uw.edu/courses/1038251/discussion_topics/3297946

This was the first email I sent. I sent a second one announcing that some students were having technical difficulties. The third email was to suggest contacting GitHub Education directly if you are still experiencing issues.
